README.md

```diff
@@ -44,7 +44,6 @@ The main configuration options are listed below:
 | SEFT_RABBITMQ_PORT                    | '5672'                         | Port for rabbit mq 1
 | SEFT_RABBITMQ_PORT2                   | '5672'                         | Port for rabbit mq 2
 | SEFT_CONSUMER_RABBIT_QUEUE            | `Seft.Responses`               | Incoming queue to read from
```

Comment:
And queue as these are no longer configurable by environment variables?

Reply:
Agreed!  Removed now